conf/sender.cfg

```diff
@@ -40,6 +40,10 @@ destination:
 
 # Outgoing messages will be read and removed from this directory.
 path: /var/spool/apel/outgoing
+# If 'path_type: directory' is supplied, the supplied 'path' will be
+# treated as if it is a directory rather than a dirq. As a result,
+# 'path' cannot contain subdirectories.
+#path_type: directory
```

Comment:
This should be uncommented but default to 'dirq'. Existing users won't get their config overwritten. The description will need updating to clarify both options.

Reply:
changed

Reply:
Also this should be first option in the messaging section.

Reply:
first option in the messaging section overall, or just before `path`?